data-prepper-plugins/dynamodb-source/src/test/java/org/opensearch/dataprepper/plugins/source/dynamodb/stream/StreamSchedulerTest.java

```diff
@@ -124,10 +124,6 @@ public void test_normal_run() throws InterruptedException {
         executorService.shutdown();
         future.cancel(true);
 
-        // Should acquire the stream partition
-        verify(coordinator).acquireAvailablePartition(StreamPartition.PARTITION_TYPE);
-        // Should start a new consumer
-        verify(consumerFactory).createConsumer(any(StreamPartition.class), eq(null), any(Duration.class));
```

Comment:
Will create an issue for fixing these tests and improving test coverage overall for the ddb source

Reply:
If you use Armeria's `Backoff` (or write a different class for backoff), you could mock the back-off such that it works for testing as you need.